pkg/loop/internal/pb/plugin_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb";
+
+package loop;
+
+service PluginService {
+  rpc NewPluginServiceFactory (NewPluginServiceFactoryRequest) returns (NewPluginServiceFactoryReply) {}
```

Comment:
is this stuttering or is the factory generating a different kind of service than the PluginService itself?

wdyt of `NewFactory`?

Reply:
This was subsequently renamed to NewReportingPluginFactory since that's what we use everywhere else